tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -966,6 +966,21 @@ handle_build() {
 	fi
 	tar tvf "${final_tarball_path}"
 
+	case ${build_target} in
+		kernel*-confidential|kernel-sev)
```

Comment:
Just to check, is the idea behind `kernel*-confidential` that we might have build_targets of kernel-confidential and kernel-<tee>-confidentials?

Reply:
Actually, I'd like to match `kernel-confidential`  and `kernel-nvidia-gpu-confidential`.

Reply:
Cool, that was just a (poor) example. Thanks for confirming